projects/aca-playwright-shared/src/fixtures/page-initialization.ts

```diff
@@ -24,28 +24,18 @@
 
 
 import { test as base } from '@playwright/test';
-import { ApiClientFactory, NodesPage, PersonalFilesPage } from '../';
+import { NodesPage, PersonalFilesPage } from '../';
 
 interface Pages {
   personalFiles: PersonalFilesPage;
   nodesPage: NodesPage;
 }
 
-interface Api {
-  superAdminApiClient: ApiClientFactory;
-}
-
-export const test = base.extend<Pages & Api>({
+export const test = base.extend<Pages>({
   personalFiles: async ({ page }, use) => {
     await use(new PersonalFilesPage(page));
   },
   nodesPage: async ({ page }, use) => {
     await use(new NodesPage(page));
-  },
-  // eslint-disable-next-line no-empty-pattern
-  superAdminApiClient: async ({ }, use) => {
```

Comment:
I'm not sure about this change, if you need to authenticate as other user as well I think you can just add username as param in this method and use it

Reply:
My understanding was that we could not use the most powerful admin user during local run. I was not focusing on adding other user at all